Index: Tabels(DataDefinitionLanguage)/tb_spacecraft_options.sql
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>CREATE TABLE DB_UNIVERSE.void.tb_spacecraft_options\n(\n    name_id VARCHAR(255) PRIMARY KEY NOT NULL,\n    create_date DATETIME2 NOT NULL, -- время созданя строки\n    store_mass INT NULL, -- максимальный вес склада\n    store_height INT NULL,\n    store_depth INT NULL,\n    store_width INT NULL,\n    radio_telescope BIT, -- делать сканирование ообектов\n    space_telescopes BIT, -- делать фото ообектов\n\n    FOREIGN KEY (name_id) REFERENCES DB_UNIVERSE.void.tb_spacecraft(name_id)\n        ON DELETE CASCADE\n        ON UPDATE CASCADE -- id планеты, -- имя космического апарата\n\n)
===================================================================
diff --git a/Tabels(DataDefinitionLanguage)/tb_spacecraft_options.sql b/Tabels(DataDefinitionLanguage)/tb_spacecraft_options.sql
--- a/Tabels(DataDefinitionLanguage)/tb_spacecraft_options.sql	
+++ b/Tabels(DataDefinitionLanguage)/tb_spacecraft_options.sql	
@@ -1,16 +1,15 @@
 CREATE TABLE DB_UNIVERSE.void.tb_spacecraft_options
 (
     name_id VARCHAR(255) PRIMARY KEY NOT NULL,
-    create_date DATETIME2 NOT NULL, -- время созданя строки
+    create_date DATETIME2 NOT NULL, -- время создания строки
     store_mass INT NULL, -- максимальный вес склада
     store_height INT NULL,
     store_depth INT NULL,
     store_width INT NULL,
-    radio_telescope BIT, -- делать сканирование ообектов
-    space_telescopes BIT, -- делать фото ообектов
+    radio_telescope BIT, -- система сканирование ообектов
+    space_telescopes BIT, -- фотоапарат
 
     FOREIGN KEY (name_id) REFERENCES DB_UNIVERSE.void.tb_spacecraft(name_id)
         ON DELETE CASCADE
-        ON UPDATE CASCADE -- id планеты, -- имя космического апарата
-
+        ON UPDATE CASCADE
 )
\ No newline at end of file
Index: Tabels(DataDefinitionLanguage)/tb_spacecraft.sql
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>CREATE TABLE DB_UNIVERSE.void.tb_spacecraft\n(\n    name_id VARCHAR(255) NOT NULL, -- имя космического апарата\n    create_date DATETIME2 NOT NULL, -- время созданя строки\n    pilot_id INT NULL, -- id пилота\n    planet_id INT NULL,\n    photo_spacecraft_id INT NULL, -- фото привезеные кароблем\n    scan_spacecraft_id INT NULL, -- фото привезеные кароблем\n    material_spacecraft_id INT NULL, -- матералы привезеные кароблем\n    start_flight_time DATETIME2 NULL, -- дата вылета коробля\n    finish_flight_time DATETIME2 NULL, -- дата возвращениея коробля\n    connection_spacecraft BIT, -- наличие связи с короблем\n        FOREIGN KEY (planet_id) REFERENCES DB_UNIVERSE.void.tb_planets(id)\n        ON DELETE CASCADE\n        ON UPDATE CASCADE -- id планеты\n)
===================================================================
diff --git a/Tabels(DataDefinitionLanguage)/tb_spacecraft.sql b/Tabels(DataDefinitionLanguage)/tb_spacecraft.sql
--- a/Tabels(DataDefinitionLanguage)/tb_spacecraft.sql	
+++ b/Tabels(DataDefinitionLanguage)/tb_spacecraft.sql	
@@ -1,16 +1,0 @@
-CREATE TABLE DB_UNIVERSE.void.tb_spacecraft
-(
-    name_id VARCHAR(255) NOT NULL, -- имя космического апарата
-    create_date DATETIME2 NOT NULL, -- время созданя строки
-    pilot_id INT NULL, -- id пилота
-    planet_id INT NULL,
-    photo_spacecraft_id INT NULL, -- фото привезеные кароблем
-    scan_spacecraft_id INT NULL, -- фото привезеные кароблем
-    material_spacecraft_id INT NULL, -- матералы привезеные кароблем
-    start_flight_time DATETIME2 NULL, -- дата вылета коробля
-    finish_flight_time DATETIME2 NULL, -- дата возвращениея коробля
-    connection_spacecraft BIT, -- наличие связи с короблем
-        FOREIGN KEY (planet_id) REFERENCES DB_UNIVERSE.void.tb_planets(id)
-        ON DELETE CASCADE
-        ON UPDATE CASCADE -- id планеты
-)
\ No newline at end of file
